packages/actions/reload-configuration/src/modules/configuration/sagas.spec.js

```diff
@@ -0,0 +1,125 @@
+import {expectSaga} from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import {throwError} from 'redux-saga-test-plan/providers'
+import {all, takeEvery} from 'redux-saga/effects'
+import {rest, externalEvents} from 'tocco-app-extensions'
+import {cache} from 'tocco-util'
+
+import * as actions from './actions'
+import rootSaga, * as sagas from './sagas'
+
+describe('reload-configuration', () => {
+  describe('modules', () => {
+    describe('configuration', () => {
+      describe('sagas', () => {
+        test('should fork child sagas', () => {
+          const generator = rootSaga()
+          expect(generator.next().value).to.deep.equal(all([takeEvery(actions.FETCH_DATA, sagas.fetchData)]))
```

Comment:
todo: when having both `takeEvery` in the same array, then here you have to add the second as well..